tools/cldr-unittest/src/org/unicode/cldr/unittest/TestCoverageLevel.java

```diff
@@ -88,7 +93,7 @@ public void oldTestInvariantPaths() {
         SupplementalDataInfo sdi = SupplementalDataInfo
             .getInstance(CLDRPaths.DEFAULT_SUPPLEMENTAL_DIRECTORY);
 
-        Set<String> allPaths = new HashSet<String>();
+        Set<String> allPaths = new HashSet<>();
```

Comment:
Some changes are just Eclipse fixing formatting. Recommend Hide whitespace differences in Gear menu.